BitmarkRegistry/Services/AssetFileService.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AssetFileService.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 7/1/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import Foundation
+import BitmarkSDK
+
+class AssetFileService {
+
+  // MARK: Properties
+  var owner: Account
+  var assetId: String
```

Comment:
I don't see we need to modify this value anywhere outside `init` method. Why do we need `var` here?

Reply:
my bad; thanks for pointing it out :)